data/data/config.tf

```diff
@@ -123,3 +123,12 @@ Should the cluster be created with ipv6 networking.
 EOF
 
 }
+
+variable "credentials_mode" {
+  type = string
+  default = null
```

Comment:
Another idea I considered is having three boolean values to represent mint, manual, passthrough.